docs/data/material/components/init-color-scheme-script/init-color-scheme-script.md

```diff
@@ -0,0 +1,137 @@
+---
+productId: material-ui
+title: InitColorSchemeScript component
+components: InitColorSchemeScript
+githubSource: packages/mui-material/src/InitColorSchemeScript
+---
+
+# InitColorSchemeScript
+
+<p class="description">The InitColorSchemeScript component eliminates dark mode flickering in server-side-rendered applications.</p>
+
+## Introduction
+
+The `InitColorSchemeScript` component is used to remove the dark mode flicker that can occur in server-side-rendered (SSR) applications.
+This script that runs before React to attach an attribute based on the user preference so that the correct color mode is applied on first render.
```

Comment:
```suggestion
This script runs before React to attach an attribute based on the user preference so that the correct color mode is applied on first render.
```